pkg/expr/expr.go

```diff
@@ -150,9 +151,14 @@ func newConstant(v string, vType config.ValueType) (*Constant, error) {
 			return nil, err
 		}
 	default: // string
```

Comment:
Update expr to support config.Duration should be a separate PR, and should really add *all the missing config types that need to be hand parsed*
like duration, email, etc.